docs/en/guides/pipeline-integrations/administration/setting-python-3-desktop.md

```diff
@@ -1,21 +1,23 @@
 ---
 layout: default
-title: Setting Python 3 for Desktop
+title: Setting Python 2 for Desktop
 pagename: setting-python-3-desktop
 lang: en
 ---
 
-# Set Python 3 as the default Python version in {% include product %} Desktop
+# Set Python 2 as the default Python version in {% include product %} Desktop
+
+**Note:** Python 2 is getting removed by November 1st 2022. You can find the notification [here](https://community.shotgridsoftware.com/t/important-notice-upcoming-removal-of-python-2-7-and-3-7-interpreter-in-shotgrid-desktop/15166).
 
 - [Windows](#windows)
 - [MacOS](#macos)
 - [CentOS 7](#centos-7)
 
 ## Windows
 
-### Manually Set the `SHOTGUN_PYTHON_VERSION` environment to 3 on Windows
+### Manually Set the `SHOTGUN_PYTHON_VERSION` environment to 2 on Windows
 
-- On the Windows taskbar, right-click the Windows icon and select **System**, navigate through the **Control Panel/System and Security/System**. 
+- On the Windows taskbar, right-click the Windows icon and select **System**, navigate through the **Control Panel/System and Security/System**. 
 
 ![](images/setting-python-3-desktop/01-setting-python-3-desktop.png)
 
```

Comment:
Line 36 value should be 2
- For the **Variable name**, add `SHOTGUN_PYTHON_VERSION`, and set the **Variable value** to `3`.